REFERENCE.md

```diff
@@ -144,6 +144,14 @@ Default policy for the INPUT chain
 
 Default value: 'DROP'
 
+##### `input_drop_invalid_packets_with_conntrack`
+
+Data type: `Boolean`
+
+Enable/Disable the `mod conntrack ctstate INVALID DROP` statement. Only works if `$disable_conntrack` is `false`. You can set this to false if your policy is DROP. This only effects the INPUT chain.
+
+Default value: `false`
```

Comment:
With `puppet-string` and autogenerated `REFERENCE.md` i dislike write the default value manually, because the manual value can be not synced with what is in the code.

Reply:
well that was your initial PR that introduced that :P I will update the init.pp again to revert it to the initial docs

Reply:
updated it again